Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -293,6 +367,110 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         XCTAssertTrue(applePayButton.waitForExistence(timeout: 3.0))
         XCTAssertTrue(applePayButton.isSelected)
     }
+    
+    func testSelection() {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.mode = .paymentWithSetup
+        settings.uiStyle = .paymentSheet
+        settings.customerKeyType = .legacy
+        settings.formSheetAction = .continue
+        settings.customerMode = .new
+        loadPlayground(app, settings)
```

Comment:
We should add a test, which tests that selection disappears when a payment method disappears.
e.g.
1. Have mobilepay selected w/ payment mode.
2. Switch to payment+sfu (which forces an update)

Ensure that mobile pay is not the selected payment method, and it is not showing on screen.

This is important because the init code for `EmbeddedPaymentMethodsView` is a bit complex -- it's easy to think  we can just set `self.selection = initialSelection` prior to calling `super.init(frame: .zero)` but this would result in showing that the unavailable payment method is still available (despite it not being selected)